### PR TITLE
Add microvm  clade

### PR DESCRIPTION
**Pull request by GTrunSec**
solved #59

### CODE REVIEW

**Comment by blaggacao** [comment]
We need to figure out if we also want to add `lib` wrappers for: https://github.com/astro/microvm.nix/tree/main/nixos-modules/microvm

File: cells/std/lib/fromMicrovmWith.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  cell,
+}: let
+  microvm = inputs.std.std.lib.fromMicrovmWith inputs;
+in {
+  # microvm <module>
+  task = microvm ({ pkgs, lib, ... }: { networking.hostName = "microvms-host";});
+}
+```
+
+_Some refactoring of the tasks may be necessary. Let the error messages be your friend._
```
Comment (blaggacao) [inline cells/std/lib/fromMicrovmWith.md:19]: this looks like out of place

File: docs/reference/clades/microvms-clade.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Microvms
```
Comment (blaggacao) [inline docs/reference/clades/microvms-clade.md:1]: this file need referencing in `SUMMARY.md`

**Comment by GTrunSec** [comment]
@blaggacao Could you help to adjust this doc? It's really hard for my editor.

**Review by blaggacao — APPROVED** [review]
LGMT

**Pull request merged**